unit_tests/test_version_utils.py

```diff
@@ -19,20 +22,23 @@
 
 class TestVersionUtils(unittest.TestCase):
     def test_01_get_branch_version_from_list(self):
-        self.assertEqual(get_branch_version_from_list(DEB_URL), '2019.1.1')
+        _ = [self.assertEqual(url_version, '2019.1.1') for url_version in get_branch_version_from_list(DEB_URL)]
```

Comment:
if the list comes back empty, this test would pass...

Reply:
Because the method `get_branch_version` checking if the file with extension  `list` or `repo`